packages/commerce-sdk-react/src/auth/storage.ts

```diff
@@ -48,7 +48,7 @@ export class CookieStorage extends BaseStorage {
     }
     set(key: string, value: string, options?: Cookies.CookieAttributes) {
         const suffixedKey = this.getSuffixedKey(key)
-        Cookies.set(suffixedKey, value, {...options, secure: true})
```

Comment:
Having `secure:true` makes cookies unreadable in a non `https` setup, which our jest jsdom isn't `https`, we need to investigate how to keep the `secure` flag but updating our testing JSDOM environment to be `https`.

Reply:
@shethj please don't merge this PR without the issue ^ addressed.

Reply:
we fixed this by setting jsdom in a secure context.